src/components/MapFacetContents.js

```diff
@@ -200,14 +200,24 @@ class MapFacetContents extends React.Component<MapFacetContentsDefaultProps, Map
 
       const points = this.props.buckets.map((bucket) => {
         const value = bucket.value; // JSON.parse(bucket.value);
+        let longitude = 0;
+        let latitude = 0;
+        if (typeof value === 'string') {
+          const valueArr = value.split(',');
+          longitude = valueArr[0];
+          latitude = valueArr[1];
+        } else {
+          longitude = value.longitude;
+          latitude = value.latitude;
+        }
         // Keep track of the boundaries of the coordinates
         return (
           <Marker
-            coordinates={[value.longitude || 0, value.latitude || 0]}
+            coordinates={[longitude || 0, latitude || 0]}
```

Comment:
If you're already validating that the latitude and longitude are valid, you shouldn't need the `|| 0` here...